InnerEye/ML/deep_learning_config.py

```diff
@@ -590,6 +590,14 @@ class TrainerParams(param.Parameterized):
         param.Boolean(default=False,
                       doc="Controls the PyTorch Lightning flag 'find_unused_parameters' for the DDP plugin. "
                           "Setting it to True comes with a performance hit.")
+    pl_limit_train_batches: Optional[int] = \
```

Comment:
what happens if the user specifies zero? Would it skip train/val automatically, is it tested?